packages/nuxt/src/core/nitro.ts

```diff
@@ -207,6 +207,7 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
 
   // Resolve user-provided paths
   nitroConfig.srcDir = resolve(nuxt.options.rootDir, nuxt.options.srcDir, nitroConfig.srcDir!)
+  nitroConfig.ignore = resolveIgnorePatterns(nitroConfig.srcDir)
```

Comment:
Shouldn't we extend/merge if user also provides `nitro.ignore`? (thinking there might be cases people only need nitro ignore patterns)

Reply:
oh, quite right 👍